src/app/shared/date.service.ts

```diff
@@ -73,4 +73,22 @@ export class DateService {
       return 'n/a';
     }
   }
+
+  getISO8601FormatFromDate(timestamp: Date): string {
+    if (timestamp) {
+      const tsAsDate = new Date(timestamp);
+      return tsAsDate.toISOString();
+    } else {
+      return 'n/a';
```

Comment:
Suggest returning null and updating method signature to so indicate.

Reply:
changed methods to return `string | null`